fortuna/src/config/run.rs

```diff
@@ -27,4 +31,17 @@ pub struct RunOptions {
     #[arg(long = "provider")]
     #[arg(env = "FORTUNA_PROVIDER")]
     pub provider: Address,
+
+    /// Path to a file containing a 20-byte (40 char) hex encoded Ethereum private key.
+    /// This key is required to submit transactions for entropy callback requests.
+    /// This key should not have any specific privileges.
```

Comment:
"specific priviledges" is a bit vague. Suggest saying "This key should not be a registered provider"